### ISSUE TITLE
Docker: Error - [pool www] please specify user and group other than root

### ISSUE THREAD

**Issue by BzowK** [Monitorr/Monitorr#189]
## Your environment:
- **Operating system**: Docker Atomic Host (Fedora 28)
- **Webserver**:       (Included in container)
- **PHP Version**:    (Included in container)
- **Monitorr Version**:   (Included in container)

## Describe your issue:
Hey Guys - 

I'm trying to install Monitorr into a Docker container, but having a small issue.  Once I create the container and start it, everything works fine until the end of it loading where I get the following two errors looping indefinitely:
- ERROR: FPM initialization failed
- ERROR: [pool www] please specify user and group other than root

I read a bit on the error and found it deals with php-fpm which doesn't allow running as root by default.  Unfortunately, I host Docker using Atomic so need to run it as root.  I found that launching php-fpm with an "-R" parameter allows it to launch as root, however; despite looking through many config files in the container cannot find the string it launches which I need to edit.

To note, I also tried editing /etc/php7/php-fpm.d/www.conf and changing the default "user" and "group" from "abc" to "root", but that did not resolve the issue or seem to make a difference.  I was hoping to find a way to configure one of the inis instead of using the -R parameter, but seems that isn't an option.

Could someone please advise as to how to make it use the -R parameter when the container is started and/or update the container if it applies?  Below is the string I used to build the container which should also provide some more detail.  

Thank You!

## String I Used to Create Container
_(Port 80 was already in use by another container so used a random one)_
docker run \
    --name=monitorr \
    --restart=on-failure \
    -v ~/monitorr/app:/app:z \
    -e TZ=America/Chicago \
    -e PGID=0 \
    -e PUID=0 \
    -p 7979:80 \
    monitorr/monitorr

_...and yes, I have read the t-shooting Wiki at: https://github.com/Monitorr/Monitorr/wiki/06-Troubleshooting :)_

**Comment by si0972** [comment]
Running dockers as root is not recommended (It can also be very dangerous). Wich branch of atomic are you using?

**Comment by BzowK** [comment]
Thanks for the reply and apologies for the late one.  I just had time to revisit today.

I'm using Atomic (Fedora 28) which is install;ed to a dedicated physical system.  I run all containers as root since it's only used in my home lab.  How would I go about getting it to work, then?  I'd assume recreating the container using different PGID & PUID but honestly don't know exact steps to create a user to get these if that's what needed.

Thanks!

**Comment by BzowK** [comment]
Nevermind - I was able to figure it out - not too difficult.  Below are steps if anyone else needs them using "newuser" as the username of the user being added...
1. SSH to Docker host as root
2. Execute "useradd newuser"
3. Execute "passwd newuser" & enter password - it must match current cryptography settings.  If it doesn't, try a different password until you get the reply "passwd: all authentication tokens updated successfully."
4. Using a different session, connect to host logging in as newly created user
5. Execute "id $user" and note the reply
6. Log out of newuser session

Once done, I simply deleted the Monitorr container which had the issue, (via Portainer) edited the string I used to originally add the container in my original post by replaced the 2 lines below...
-e PGID=0 
-e PUID=0 
with what I noted from step #5 above...
-e PGID=1001
-e PUID=1001

I then pasted the updated string into my SSH session still logged in as root.  After a few seconds, I could successfully access the initial web page of Monitorr.

Thanks!
